tests/main/session-tool/task.yaml

```diff
@@ -56,11 +67,17 @@ restore: |
 
     # Restart background stuff we stopped.
     sh defer.sh && rm -f defer.sh
+
 debug: |
-    echo "Active sessions:"
-    for session_id in $(loginctl list-sessions --no-legend | awk '{ print($1) }'); do
-        echo "Details of session $session_id"
-        loginctl show-session "$session_id"
-    done
+    (
+        echo "Active sessions:"
+        for session_id in $(loginctl list-sessions --no-legend | awk '{ print($1) }'); do
+            echo "Details of session $session_id"
+            loginctl show-session "$session_id"
+        done
```

Comment:
You could move this to a lib/ shell helper and avoid duplication, but not a blocker.

Reply:
I can do this in a follow-up.

Reply:
Opened as #8448